lix/packages/sdk/src/open/openLix.ts

```diff
@@ -189,7 +187,25 @@ export async function openLix(args: {
 			}
 			return addComment({ ...args, db, currentAuthor });
 		},
+		createBranch: (
+			name: string,
+			{ branchId }: { branchId: string | undefined } = { branchId: undefined },
+		) =>
+			createBranch({
+				db,
+				name: name,
+				branchId,
+			}),
+		switchBranch: ({ branchId, name }: { branchId?: string; name?: string }) =>
```

Comment:
It's reasonable to expect that an app knows the id of a branch. 

- simplifies the API
- fixes a bug with the type (brnachId and name can't be both undefined)

```suggestion
		switchBranch: (args: { id: string }) =>
```